util/chunk/chunk.go

```diff
@@ -725,3 +725,34 @@ func (c *Chunk) AppendPartialRows(colOff int, rows []Row) {
 		}
 	}
 }
+
+// AppendRowsByColIdxs appends multiple rows by its colIdxs to the chunk.
+// 1. every columns are used if colIdxs is nil.
+// 2. no columns are used if colIdxs is not nil but the size of colIdxs is 0.
+func (c *Chunk) AppendRowsByColIdxs(rows []Row, colIdxs []int) (wide int) {
```

Comment:
Please add unit test cases for `AppendRowsByColIdxs` & `AppendPartialRowsByColIdxs`.

Reply:
Todo

Reply:
@pingyu PTAL